angr/knowledge_plugins/cfg/cfg_model.py

```diff
@@ -99,13 +100,20 @@ def _get_cmsg(cls):
         return cfg_pb2.CFG()
 
     def serialize_to_cmessage(self):
+        if "Emulated" in self.ident:
+            l.error("Serializing a CFGEmulated instance is currently not supported.")
```

Comment:
I would raise a `NotImplementedError("Serializing a CFGEmulated instance is currently not supported.")` instead.